src/Microsoft.Health.Fhir.Core/Exceptions/JobNotFoundException.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Hl7.Fhir.Model;
+
+namespace Microsoft.Health.Fhir.Core.Exceptions
+{
+    public class JobNotFoundException : FhirException
```

Comment:
I think these exceptions could go into Operations folder since it's operations specific.